### ISSUE TITLE
Port IncludeExtensions

### ISSUE THREAD

**Issue by brandongregoryscott** [AndcultureCode/AndcultureCode.CSharp.Extensions#25]
_NOTE: Must be an andculture employee to work on this issue due to requiring access to private resources._

From the LMS project, port over the `IncludeExtensions` static class used for building out include property strings. While it has unit tests, please review during the port and see if it requires more.

Once done, we will want to refactor this and other projects to leverage the new extension method approach.

**Comment by joshuapeters** [comment]
@brandongregoryscott I'll take this one as well 👍

**Comment by joshuapeters** [comment]
@brandongregoryscott @wintondeshong Upon looking into this, the IncludeExtensions in the LMS projects are off of our home-grown Include and IncludeBase models.

Am I safe to proceed by adding a task to the Core repo for those models, completing it, and then coming back to this task? If not please let me know what I should do to move forward.

**Comment by wintondeshong** [comment]
@joshuapeters yeah, I'm okay with moving it to Core so long as it didn't bring EF along for the ride. Maybe my version of IncludeExtensions, Include and IncludeBase are old. I don't see any dependency on EF at least in those 3 classes. Mind sharing a snapshot of the file names that would all be ported?
